pom.xml

```diff
@@ -14,11 +14,6 @@
   <name>Jenkins promoted builds plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Promoted+Builds+Plugin</url>
 
-  <properties>
-    <jenkins.version>1.565</jenkins.version>
```

Comment:
The version and java specification should stay explicit. Feel free to bump it if 1.565 is not enough for your case